### DISCUSSION TITLE
Buttons for time forwarding

### DISCUSSION THREAD

**Discussion by Battenhatch** [vigoren/foundryvtt-simple-calendar#263] [Q&A] [answered]
Hi

I'm looking at the buttons that turn the time to a certain time of day (Dawn, Midday, Dusk and Midnight) ... and three of those alle seem to forward the time to the net midnight. Only Midday doesn't do that at my end. I've tried to find a setting for this, but I'm coming up blank. Is this as it should be? Can it be changed somewhere?

**Comment by vigoren** [comment, marked as answer]
Hey!

To figure out when dawn and dusk are those buttons rely on knowing when the sunrise and sunset times are. To set up the sunrise and sunset times you can open up the configuration for Simple Calendar, go to the Year Settings and under seasons set them up. If there are no seasons you can add one and configure it.
Hope that helps

**Comment by Battenhatch** [comment]
I knew there was something I was missing - thanks :)
